third_party/tests/Verilator/Verilator.log

```diff
@@ -4863,7 +4863,7 @@ Processing: -cd . +incdir+. -parse -noelab -nocache -nobuiltin -nonote -noinfo -
 Processing: -cd . +incdir+. -parse -noelab -nocache -nobuiltin -nonote -noinfo -sverilog -timescale=1ns/1ns t_interface_down_gen.v -l t_interface_down_gen.v.log
 [ERR:PP0102] t_interface_down_gen.v:75:1: Unknown macro "error".
 
-[SNT:PA0207] t_interface_down_gen.v:75:52: Syntax error: no viable alternative at input 'Commercial sims choke',
+[SNT:PA0207] t_interface_down_gen.v:75:52: Syntax error: missing '(' at 'choke',
```

Comment:
Do we agree on a real syntax issue here ?

Reply:
Yes, I'm perfectly fine having a Syntax error here.